CHANGELOG.md

```diff
@@ -69,6 +69,10 @@ changes.
 
 -  `hydra-cluster` executable can be used to provide a local cardano "network" with `--devnet` argument
 
+- Introduce new `Hydra.Cardano.Api.Pretty` module in `hydra-cardano-api` and
+  move functions `renderTx`, `renderTxWithUTxO` and `renderTxs` from
+  `hydra-node` package to this new module.
```

Comment:
There is already a section about `hydra-cardano-api` changes IIRC.

Should: group relevant changes together.